Dockerfile

```diff
@@ -12,20 +12,22 @@ RUN apt-get update -qqy && DEBIAN_FRONTEND=noninteractive apt-get install -qqy -
     libqt5gui5 \
     libqt5widgets5 \
     libqt5printsupport5 \
-    libicu57 \
-    libmpfr4 \
+    libicu63 \
+    libmpfr6 \
 && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
 
-ARG VERSION=2.1.7
-ARG FILE=MiniZincIDE-$VERSION-bundle-linux-x86_64
+ARG VERSION=2.3.1
+ARG FILE=MiniZincIDE-$VERSION-bundle-linux
 
 RUN cd /tmp \
-    && curl -LOSs https://github.com/MiniZinc/MiniZincIDE/releases/download/$VERSION/$FILE.tgz \
-    && tar -xzf $FILE.tgz \
+    && curl -LOSs https://github.com/MiniZinc/MiniZincIDE/releases/download/$VERSION/$FILE-x86_64.tgz \
+    && tar -xzf $FILE-x86_64.tgz \
     && (cd $FILE \
-     && cp minizinc flatzinc fzn-* mzn* solns2* /usr/local/bin \
+     && cp bin/minizinc* bin/fzn-* bin/findMUS bin/mzn2doc /usr/local/bin \
      && cp -r share/minizinc /usr/local/share/ \
+     && cp -r lib /opt/minizinc \
```

Comment:
`/usr/local/lib/` would be better as the binaries are in `/usr/local/bin`. In addition, it's also in ld loading paths.

Reply:
Done